olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```
@@ -214,7 +214,7 @@ spec:
             displayName: Version
             path: version
         version: v1beta1
-      - description: Represents an installation of a particular version of Knative Kafka components
+      - description: Additionl extensions based on Apache Kafka for production-ready Knative Eventing deployments
```

what do you think about this instead?
```suggestion
      - description: Knative Kafka is the Knative Eventing extensions for Apache Kafka. It allows to trigger services based on Apache Kafka topics and create your event-driven applications with a simple HTTP interface with the the underlying reliability and efficiency of Apache Kafka.
```

---

that's quite a mouthful. How about:

> Knative Kafka is the Knative Eventing extensions for Apache Kafka. Create event-driven applications with a simple HTTP interface while leveraging the underlying reliability and efficiency of Apache Kafka